### DISCUSSION TITLE
Type error when building generated actions with tsup

### DISCUSSION THREAD

**Discussion by MichaelGrinstead** [wevm/wagmi#3470] [Q&A]
I'm having an issue with typescript being unable to infer the types of the generated actions, after generating the actions and running tsup I get this error 

`error TS2742: The inferred type of 'myFunction' cannot be named without a reference to '.pnpm/@wagmi+core@2.2.1_react@18.2.0_typescript@5.2.2_viem@2.1.1/node_modules/@wagmi/core/dist/types/actions/codegen/createReadContract'. This is likely not portable. A type annotation is necessary.`

Thanks

**Comment by tmm** [comment]
Feel free to create an issue with minimal reproduction attached.

**Reply by DonFungible** [reply]
Adding `overridePackageName: "@wagmi/core"` solved this for me

(whoops responded in the wrong section, can't delete comment)
